zookeeper-server/src/main/java/org/apache/zookeeper/cli/RestoreCommand.java

```diff
@@ -119,6 +126,7 @@ private OptionFullCommand() {
     options.addOption(new Option(OptionShortForm.LOCAL_RESTORE_TEMP_DIR_PATH, true,
         OptionLongForm.LOCAL_RESTORE_TEMP_DIR_PATH));
     options.addOption(new Option(OptionShortForm.DRY_RUN, false, OptionLongForm.DRY_RUN));
+    options.addOption(new Option(OptionShortForm.OVERWRITE, false, OptionLongForm.OVERWRITE));
```

Comment:
The classes `OptionFullCommand`, `OptionLongForm`, `OptionShortForm` are not redundant because the strings in these classes are all referred several times. And since we have many options (10 for now), it is easier to keep track and easier to maintain. Also, something special about this command is that its parsing logic is in another class instead of `RestoreCommand` (to be consistent with the OSS patch), making these constant strings in a public class makes it easier to refer from another class too.
getUsageStr() will show up to the user every time there's something wrong when parsing the command, and it includes detailed explanation and usage method of each option, so user can modify their command accordingly. And when passing in "-h" option, the usageStr will also show up. This is a feature requested by our user, so I won't say it's redundant.